test/preview/document_stores/test_memory.py

```diff
@@ -12,3 +15,123 @@ class TestMemoryDocumentStore(DocumentStoreBaseTests):
     @pytest.fixture
     def docstore(self) -> MemoryDocumentStore:
         return MemoryDocumentStore()
```

Comment:
I'd take the chance to ditch this fixture and instance the doc store in each test.

Reply:
I would, but it is part of the design pattern from the superclass test.

Reply:
Ah ok. Must have missed that. 👍

Reply:
Yep that instance is used on test of all the base docstore features, like filtering, writing, etc etc